### DISCUSSION TITLE
STP file of the hardware assembly

### DISCUSSION THREAD

**Discussion by KytorIndustries** [sanni/cartreader#706] [General Chat]
I am designing an enclosed shell for HW5. The "3dmodel.stl" is putting up a fight being repaired by various software (Prusa, Meshmixer, FreeCAD, etc). Is it possible to get the STP file?

**Comment by sanni** [comment]
Here are the source step/dxf files. From there on everything gets messy since I still need to learn a proper 3D modeling software.

[Adafruit Si5351A.step.zip](https://github.com/sanni/cartreader/files/10715755/Adafruit.Si5351A.step.zip) (exported using KiCAD from the [PCB file](https://github.com/adafruit/Adafruit-Si5351A-Clock-Generator-Breakout-PCB))
[Main PCB Stand - Right N64 Adapter.step.zip](https://github.com/sanni/cartreader/files/10715756/Main.PCB.Stand.-.Right.N64.Adapter.step.zip) (Made by [ducky92](https://github.com/sanni/cartreader/blob/master/hardware/main_pcb_hw5/Main%20PCB%20Stand%20-%20Right%20N64%20Adapter.step))
[main_pcb.step.zip](https://github.com/sanni/cartreader/files/10715757/main_pcb.step.zip)  (exported using KiCAD from the [PCB file](https://github.com/sanni/cartreader/blob/master/hardware/main_pcb_hw5/main_pcb_hw5.kicad_pcb))
[MegaPro.step.zip](https://github.com/sanni/cartreader/files/10715758/MegaPro.step.zip)  (exported using KiCAD from the [robotdyn library file](https://robotdyn.com/pub/media/lib/robotdyn.lbr))
[MKS MINI12864 V3.0_002.dxf.zip](https://github.com/sanni/cartreader/files/10715759/MKS.MINI12864.V3.0_002.dxf.zip) (from the [Makerbase Github](https://github.com/makerbase-mks/MKS-MINI12864-V3/tree/main/hardware/MKS%20MINI12864%20V3.0_002))
[six_slot_adapter.step.zip](https://github.com/sanni/cartreader/files/10715760/six_slot_adapter.step.zip)  (exported using KiCAD from the [PCB file](https://github.com/sanni/cartreader/blob/master/hardware/six_slot_adapter/six_slot_adapter.kicad_pcb))
[snespic_adapter.step.zip](https://github.com/sanni/cartreader/files/10715761/snespic_adapter.step.zip)  (exported using KiCAD from the [PCB file](https://github.com/sanni/cartreader/blob/master/hardware/cic_adapter/PIC_adapter.kicad_pcb))

**Reply by KytorIndustries** [reply]
Thanks. I ended up getting a shell mostly designed without the step files, but this will be helpful for some digital validation.
